etc/mpv-powershell-completion.ps1

```diff
@@ -0,0 +1,287 @@
+# PowerShell command line completion for the mpv media player.
```

Comment:
Move description below licence header

Reply:
I'm not very good working with git, had problems as often, I hope it's okay now.